src/main.js

```diff
@@ -68,7 +68,15 @@ window.WebVRConfig = Util.extend({
   // When set to true, this will cause a polyfilled VRDisplay to always be
   // appended to the list returned by navigator.getVRDisplays(), even if that
   // list includes a native VRDisplay.
-  ALWAYS_APPEND_POLYFILL_DISPLAY: false
+  ALWAYS_APPEND_POLYFILL_DISPLAY: false,
+
+  // There are scenarios where the native WebVR API exists, and instead of returning
```

Comment:
@tforcram Good point, I'll clarify this with specific ranges and not something that will occur Chrome M60+